src/main/java/io/dropwizard/kafka/lifecycle/TopicCreationLifecycleListener.java

```diff
@@ -0,0 +1,23 @@
+package io.dropwizard.kafka.lifecycle;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.eclipse.jetty.util.component.LifeCycle;
+
+import java.util.Collection;
+
+public abstract class TopicCreationLifecycleListener implements LifeCycle.Listener {
```

Comment:
Instead of Jetty's lifecycle, you should use Dropwizard's abstraction on top of it, `Managed`.

Reply:
Okay, thought I needed `environment.lifecycle().addListener()`

Reply:
For these type of modules, I usually prefer `environment.lifecycle().manage(...)` instead, unless there's a reason for it not to be done that way.